File: EventBus/app/src/main/java/com/lee/eventbus/model/UserInfo.java
package com.lee.eventbus.model;

/**
 * @author jv.lee
 * @date 2019-08-17
 * @description
 */
public class UserInfo {
    private String name;
    private String age;

    public UserInfo(String name, String age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getAge() {
        return age;
    }

    public void setAge(String age) {
        this.age = age;
    }

    @Override
    public String toString() {
        return "UserInfo{" +
                "name='" + name + '\'' +
                ", age='" + age + '\'' +
                '}';
    }
}

File: KotlinLibrary/app/src/main/java/com/lee/library/model/entity/UserInfo.kt
package com.lee.library.model.entity

/**
 * @author jv.lee
 * @date 2019-08-15
 * @description
 */
data class UserInfo(
    val images: List<String>,
    val info: Info
)

data class Info(
    val description: String,
    val id: Int,
    val name: String
)